packages/interactive/lib/src/workspace_code.dart

```diff
@@ -45,7 +45,7 @@ class WorkspaceCode {
     return '''
 // AUTO-GENERATED, PLEASE DO NOT MODIFY BY HAND
 
-import 'package:interactive/src/runtime_support.dart';
+//import 'package:interactive/src/runtime_support.dart';
```

Comment:
This may not be the perfect way, since we *really* need this runtime_support dart